example/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-alpine
```

Comment:
Maybe you should consider upgrading latest stable 3.8 python ?  

https://docs.python.org/3.8/whatsnew/3.8.html